cmd/kubelet/app/server_bootstrap_test.go

```diff
@@ -298,14 +298,16 @@ func (s *csrSimulator) ServeHTTP(w http.ResponseWriter, req *http.Request) {
 		ca := &authority.CertificateAuthority{
 			Certificate: s.serverCA,
 			PrivateKey:  s.serverPrivateKey,
-			Backdate:    s.backdate,
 		}
 		cr, err := capihelper.ParseCSR(csr.Spec.Request)
 		if err != nil {
 			t.Fatal(err)
 		}
 		der, err := ca.Sign(cr.Raw, authority.PermissiveSigningPolicy{
 			TTL: time.Hour,
+			Now: func() time.Time {
+				return time.Now().Add(-s.backdate)
+			},
```

Comment:
should this be setting backdate instead:
```
Backdate: s.backdate,
```

Reply:
I think this was leftover from the original code that changed the overall duration of all certs and not just short lived ones.  Fixed.